assets/queries/openAPI/link_object_incorrect_ref/query.rego

```diff
@@ -0,0 +1,35 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	link := doc.components.responses[r].links[l]
```

Comment:
I think we shouldn't check components, since a link on components can describe external links. I think this query refers to elements outside components, no?

Reply:
Since components object could have a responses object that can have a link object, I think it is necessary to check if this link object, when it is a reference, is outside field 'links' of components

Reply:
Makes sense